src/ucs/datastruct/callbackq.h

```diff
@@ -186,6 +210,43 @@ ucs_status_t ucs_callbackq_remove_safe(ucs_callbackq_t *cbq, ucs_callback_t cb,
 
 
 /**
+ * Add a callback to the slow path list.
+ * This can be used from any context and any thread, including but not limited to:
+ * - A callback can add another callback.
+ * - A thread can add a callback while another thread is dispatching callbacks.
+ *
+ * Important note: It is not allowed to add the same element more than once.
```

Comment:
@brminich we would need this only for debug version
setting list fields to null is also good option - we may do it as part of callbackq.c, not in list.h - because then would need to include config.h from list.h which means can't include list.h from api header files.